src/ngraph/pass/pass_util.cpp

```diff
@@ -0,0 +1,40 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/pass/pass_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+std::function<bool(std::shared_ptr<Node>)> ngraph::pass::get_no_fan_out_function()
+{
+    auto ret_fun = [](std::shared_ptr<Node> n) {
+        auto users = n->get_users(true);
+        std::set<std::shared_ptr<Node>> user_set(users.begin(), users.end());
```

Comment:
We should really decide whether we want to use the `std` namespace in the source files or not.  Typically that hasn't been the case, but its not a deal breaker 😉